Makefile

```diff
@@ -16,7 +16,7 @@ default: clean build
 
 ## `make preview`: start the built-in Jekyll preview
 preview: clean
-	$S export LANG=C.UTF-8 ; bundle exec jekyll serve --incremental
+	$S export LANG=C.UTF-8 ; bundle exec jekyll serve --incremental --host=0.0.0.0
```

Comment:
We can discard change, assuming it's an artifact left over from testing.

Reply:
Thanks, I've pushed an update discarding the change.